docs/parcels.md

```diff
@@ -0,0 +1,110 @@
+# Parcels
+
+A single-spa parcel is a bundle of functionality (like an application) meant to be controlled imperatively 
+by an application, Parcels use similar methodology as Applications so we recommend [starting with Applications.](/docs/applications.md)
+A parcel can be as large as an application or as small as an component and written in 
+and language as long as it exports the correct lifecycle events. In a single-spa world, your SPA contains 
+many registered applications and potentially many parcels. Usually you'll want to mount a parcel within the 
+context of an application, this will automatically unmount the parcel when the application is unmounted.
```

Comment:
I would possibly reword this sentence for clarity. Maybe something like "Typically, we recommend you mount a parcel within the context of an application because it will automatically be unmounted with the application."